irods/test/data_obj_test.py

```diff
@@ -2024,6 +2024,19 @@ def test_settings_load_and_save_471(self):
                     config.load(**load_logging_options)
                     self.assertTrue(config.data_objects.auto_close, RANDOM_VALUE - i - 1)
 
+    def test_setting_xml_parser_choice_by_environment_only__issue_584(self):
```

Comment:
To be clear - and just for my own understanding - there is assumed to be no config file present for this test, right? Should we make sure of this before we proceed in the test?

Reply:
The variable PYTHON_IRODSCLIENT_CONFIGURATION_PATH must be present in the environment for a config file to be autoloaded.  I suppose we could skip the test in that event.

Reply:
done